src/components/TasksTable/index.jsx

```diff
@@ -23,6 +23,9 @@ const sorted = pipe(
   },
   link: {
     textDecoration: 'none',
+    '&:nth-of-type(odd)': {
+      backgroundColor: '#ecffff',
```

Comment:
I think the color is somewhat fluorescent which causes it to be not so relaxing to the eye. Could we experiment with a different color? Also, I think we should define the color in theme.jsx instead of hardcoding it here.

Reply:
yup i agree, ok!